static/css/style.css

```diff
@@ -849,18 +851,27 @@ fieldset[disabled] .source-buttons .btn-primary.active {
   border: 1px solid transparent;
   white-space: nowrap;
   line-height: 1.42857143;
-  border-radius: 0px;
   -webkit-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   text-transform: uppercase;
   text-shadow: none;
-  letter-spacing: .15em;
+  letter-spacing: .05em;
   font-weight: bold;
-  font-size: .9em;
```

Comment:
I think this is also changing the font size on the documentation page - see the difference in size of the text underneath each doc icon.